src/lib/storage/buffer/page_id.hpp

```diff
@@ -0,0 +1,62 @@
+#pragma once
+#include <magic_enum.hpp>
+#include "boost/integer/static_log2.hpp"
+
+namespace hyrise {
+
+#ifdef __APPLE__
+constexpr size_t OS_PAGE_SIZE = 16384;
```

Comment:
Short comment on that? I found this site (https://developer.apple.com/library/archive/documentation/Performance/Conceptual/ManagingMemory/Articles/AboutMemory.html) which mentions the changes of page sizes for MacOS. I was totally unaware, so a short comment might help.

Reply:
Good point! Generally, I am not entirely happy with this difference between OSX and Linux. Linux may also have larger pages..